### PR TITLE
script restructuring and optimization

### PR DESCRIPTION
**Pull request by bmos**
Paul did a seat-of-the-pants rewrite of some things without testing anything (he doesn't own FG).
Then I fixed the stuff that didn't work (only a couple things).

### CODE REVIEW

File: scripts/real_encumbrance.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ end
 local function handleApplyPenaltiesArgs(node)
 	local nodePC
 	local rActor
-
+
```
Comment (bmos) [inline scripts/real_encumbrance.lua:21]: 😢

Comment (crusopaul) [reply scripts/real_encumbrance.lua:21]: sad reacts only

File: scripts/real_encumbrance.lua [hunk 2]
```diff
@@ -18,7 +18,7 @@ end
 local function handleApplyPenaltiesArgs(node)
 	local nodePC
 	local rActor
-	
+
```
Comment (bmos) [inline scripts/real_encumbrance.lua]: Thank you.

**Comment by bmos** [comment]
woohoo!

**Pull request merged**